lib/Backend/BackwardPass.cpp

```diff
@@ -2965,6 +2974,43 @@ BackwardPass::DeadStoreOrChangeInstrForScopeObjRemoval()
     return false;
 }
 
+IR::Instr *
+BackwardPass::TryChangeInstrForStackArgOpt()
+{
+    IR::Instr * instr = this->currentInstr;
+    if (instr->DoStackArgsOpt(this->func))
+    {
+        switch (instr->m_opcode)
+        {
+            case Js::OpCode::TypeofElem:
+            {
+                /*
+                    Before:
+                        dst = TypeOfElem arguments[i] <(BailOnStackArgsOutOfActualsRange)>
```

Comment:
ok, makes sense


---
In reply to: [70507186](https://github.com/Microsoft/ChakraCore/pull/1261#discussion_r70507186) [](ancestors = 70507186,70345281)